src/mlang/driver.ml

```diff
@@ -155,6 +155,13 @@ let driver (files : string list) (debug : bool) (var_info_debug : string list) (
             Bir_to_c.generate_c_program combined_program function_spec !Cli.output_file;
             Cli.debug_print "Result written to %s" !Cli.output_file
           end
+          else if String.lowercase_ascii backend = "dgfip_c" then begin
+            Cli.debug_print "Compiling the codebase to DGFiP C...";
```

Comment:
You should also modify the usage message related to the backend option in `cli.ml`.